src/base/static/components/form-fields/types/index.js

```diff
@@ -44,7 +44,7 @@ export {
 } from "../../molecules/form-field-types/number-field";
 export {
   default as NumberFieldResponse,
-} from "../../molecules/form-field-response-types/number-field-response";
+} from "../../molecules/place-detail-fields/number";
```

Comment:
@goldpbear  FYI I created a new dir `place-detail-fields`, and I noticed that it has the same purpose as everything in the `form-field-response-types` dir, so I renamed it to clarify that these components are unrelated to the form.

It seems the purpose of this dir is to hold components that render fields in our place detail view. Since these components are only responsible for rendering saved data into a place detail view, they have nothing to do with the "form". So I think we should start removing the concept of a "form" from these components, and decouple the place detail rendering from the form rendering.

Happy to chat if you have any questions.